lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -901,11 +901,20 @@ export abstract class ClientApplication {
      * Adds event callbacks to array
      * @param callback 
      */
-    addEventCallback(callback: EventCallbackFunction) {
+    addEventCallback(callback: EventCallbackFunction): string | null {
         if (this.isBrowserEnvironment) {
             this.logger.verbose("Event callback registered");
```

Comment:
Can this log message included the `callbackId`?

Reply:
Done